example/src/App.js

```diff
@@ -4,6 +4,16 @@ import React, { Component } from 'react';
 import { MapInteractionCSS } from 'react-map-interaction';
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: {
```

Comment:
I think you should flatten the state here.  `value` just makes it a bit more confusing and I don't think using the spread operator makes things simple to follow.

```js
    this.state = {
      scale: 1,
      translation: { x: 0, y: 0 }
    };
```